src/react/src/components/Contribute/ProductionLocationDialog.jsx

```diff
@@ -126,8 +119,7 @@ const ProductionLocationDialog = ({
     const fieldSetNumber = round(size(filteredAdditionalFields) / 2);
 
     const isClaimable =
-        claimStatus === PRODUCTION_LOCATION_CLAIM_STATUSES_ENUM.UNCLAIMED &&
-        moderationStatus !== MODERATION_STATUSES_ENUM.PENDING;
+        claimStatus === PRODUCTION_LOCATION_CLAIM_STATUSES_ENUM.UNCLAIMED;
```

Comment:
Please ensure that it will be unclaimable even if there is a **pending** claim, as previously required by stakeholders. Check how this works on the facility profile page when there is a claim with a 'Pending' status.
See the ticket -> https://opensupplyhub.atlassian.net/browse/OSDEV-384

Reply:
@vladsha-dev you are right, thank you for noticing this! I've applied the fix for this case, pls re-check.